### ISSUE TITLE
webmotors.com.br

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#4411]
### Basic Information

**Test link:** `https://www.webmotors.com.br/carros/estoque/audi/a3/de.2017?tipoveiculo=carros&anode=2017&marca1=AUDI&modelo1=A3`
**Category:** other
**Reported from:** Brazil
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36
**Extension:** Nano Defender 15.0.0.201

### Additional Message

Whenever you change any filter in the website, the page goes blank and I need to manually refresh using F5 or the Refresh Page button in Chrome.
Basically Nano Defender is breaking the website "ajax" reload feature (I suppose).
Without Nano Defender (using Firefox for example, without any extension) I can browse just fine in this website.

**Comment by jspenguin2017** [comment]
Can't reproduce breakage, the left side filters is working as expected for me. Placeholder fixed.
